packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.ClientModel/src/ScmCodeModelGenerator.cs

```diff
@@ -36,6 +37,7 @@ protected override void Configure()
             AddMetadataReference(MetadataReference.CreateFromFile(typeof(ClientResult).Assembly.Location));
             AddMetadataReference(MetadataReference.CreateFromFile(typeof(BinaryData).Assembly.Location));
             AddMetadataReference(MetadataReference.CreateFromFile(typeof(JsonSerializer).Assembly.Location));
+            AddTypeToKeepPublic(ModelReaderWriterContextDefinition.s_name);
```

Comment:
Can we use the Name property?

Reply:
The name property is instanced since its an override from the abstract base.